R/backend-snowflake.R

```diff
@@ -265,4 +288,16 @@ snowflake_paste <- function(default_sep) {
   }
 }
 
+snowflake_pmin_pmax_sql_expression <- function(dots, comparison){
+  dot_combined <- dots[[1]]
+  for (i in 2:length(dots)){
+    dot_combined <- snowflake_pmin_pmax_builder(dots[i], dot_combined, comparison)
+  }
+  glue_sql2(sql_current_con(), "{dot_combined}")
```

Comment:
This use of `glue_sql2()` doesn't make much sense. I guess you should rather use it in `snowflake_pmin_pmax_builder()` and simply return `dot_combined` here.

Reply:
Updated. thanks!